src/Nethermind/Nethermind.Blockchain/FullPruning/CopyTreeVisitor.cs

```diff
@@ -21,25 +23,26 @@ namespace Nethermind.Blockchain.FullPruning
     /// </remarks>
     public class CopyTreeVisitor : ITreeVisitor, IDisposable
     {
-        private readonly IPruningContext _pruningContext;
         private readonly ILogger _logger;
         private readonly Stopwatch _stopwatch;
         private long _persistedNodes = 0;
         private bool _finished = false;
         private readonly WriteFlags _writeFlags;
         private readonly CancellationToken _cancellationToken;
         private const int Million = 1_000_000;
+        private WriteBatcher _writeBatcher;
 
         public CopyTreeVisitor(
             IPruningContext pruningContext,
+            CancellationToken cancellationToken,
```

Comment:
general convention is to put cancellation token at the last parameter (as it often has default)